src/padding.rs

```diff
@@ -0,0 +1,45 @@
+use crate::CellOption;
+use papergrid::{Entity, Grid, Settings};
+
+/// Padding is responsible for a left/right/top/bottom indent of particular and fill one set characters.
+///
+/// ```rust,no_run
+///   # use tabled::{Style, Padding, Row, Table, Modify};
+///   # let data: Vec<&'static str> = Vec::new();
+///     let table = Table::new(&data).with(Modify::new(Row(..1)).with(Padding::new(0, 0, 1, 1).set_fill('>', '<', '^', 'V')));
+/// ```
+pub struct Padding(papergrid::Padding);
+
+impl Padding {
+    /// Construct's an Padding object.
+    pub fn new(left: usize, right: usize, top: usize, bottom: usize) -> Self {
+        Self(papergrid::Padding {
+            top: papergrid::Indent::default().set_size(top),
+            bottom: papergrid::Indent::default().set_size(bottom),
+            left: papergrid::Indent::default().set_size(left),
+            right: papergrid::Indent::default().set_size(right),
```

Comment:
I would actually run a `new` here in order to "control" the default `fill` field from here.

Reply:
That'd good but I would create this constant right in tabled/padding.rs.
But that's ok